build/Targets/VSL.Imports.targets

```diff
@@ -270,6 +270,23 @@
     </NuGetPackageToIncludeInVsix>
   </ItemDefinitionGroup>
 
+  <!-- Currently, the Willow installer can't install vsixes that have a <Prerequisites> element
+       but the Dev15 RC SDK packages can NOT install vsixes without them as part of the build.
+       To work around this conflict, we include them in our manifest files, but if we're building
+       with something older than Dev15 RC (as we do for official builds that we insert into setup)
```

Comment:
Is this bad if our official builds were updated to the RC? I'm not aware of an immediate plan, but just want to check that the condition is what we want it to be.